metrics/metrics.go

```diff
@@ -58,6 +59,12 @@ func New(metricsAddr string, pebbleMetricsProvider func() *pebble.Metrics) (*Met
 		return nil, err
 	}
 
+	if m.dhfindLatency, err = meter.SyncInt64().Histogram("ipni/dhstore/dhfind_latency",
```

Comment:
`ipni/dhstore/find_latency` tagged by `dh` true/false?

Reply:
A problem with doing that is that the http_latency (find_latency) is used to measure both /multihash and /metadata lookups. A dh tag would never apply to a /metadata lookup, or would always be true.

Also, the dhfind latency has a tag for whether or not it is the time to the first result. In the dh=true, time to first result would not apply since the client determines the time from the first mh lookup to the first metadata result returned by a /metadata lookup.

So, I think using two separate metrics keeps things somewhat less confusing.